app/controllers/pages_controller.rb

```diff
@@ -1,6 +1,7 @@
 class PagesController < ApplicationController
   def index
     solution = Solution.first
+
     IterationsChannel.broadcast!(solution)
```

Comment:
Unless `solution` is used more than once, this is a documenting variable.

`IterationsChannel.broadcast!(Solution.first)` is very clear without it.